_posts/2019-06-21-formats.md

```diff
@@ -0,0 +1,203 @@
+as that at which users need to access their data are -- and that across as many
+platforms as possible. We also know that, optimally, this ecosystem should all
+just keep working for years to come. But while these requirements need to be
+fulfilled, something must give.
+
+<blockquote>We think the only scalable way forward is to work together on an ever smaller
+number of formats.</blockquote>
+
+We think the only scalable way forward is to work together on an ever smaller
+number of formats. That's why we've been concentrating on open formats instead
+of adding new proprietary formats. For example, [Bio-Formats
+6.1](https://www.openmicroscopy.org/2019/05/20/bio-formats-6-1-0.html) adds
+support for the open BigDataViewer (BDV) format, a strong candidate for support
+across the community.
+
+<div class="icon_row">
+  <div class="icon_column">
+    <img style="width: 100%"
+         src="{{ site.baseurl }}/images/2019/binary-tiff.png"
+         alt="Simple TIFF icon"/>
+  </div>
+  <div class="icon_column">
+    <img style="width: 100%"
+         src="{{ site.baseurl }}/images/2019/binary-hdf.png"
+         alt="Simple HDF5 icon"/>
+  </div>
+  <div class="icon_column">
+    <img style="width: 100%"
+         src="{{ site.baseurl }}/images/2019/binary-zarr.png"
+         alt="Simple zarr icon"/>
+  </div>
+</div>
+
+BDV provides a testbed for moving beyond the current single binary format of
+OME-TIFF. The OME Model will be extended to permit describing the multiscale,
+multidimensional data that is currently stored in BDV XML/H5. As a stable
+container format, HDF5 allows us a quick way to validate these concepts.
+
+At the same time there's a consensus that HDF5 itself as currently implemented
+cannot be the only binary container for our community, and, therefore, we are
+also collaborating on next-generation open-source, chunked (or "cloud") formats
+for the scale of data generated by future acquisition systems. Two candidates —
+[Zarr](http://zarr-developers.github.io) and
+[N5](https://github.com/saalfeldlab/n5) — were independently developed but
+overlap in most of their core concepts. Both communities have since begun
+[work](https://zarr-developers.github.io/zarr/specs/2019/06/19/zarr-v3-update.html)
+on a common storage spec, and other groups from
+[NetCDF](https://www.unidata.ucar.edu/blogs/news/entry/netcdf-and-native-cloud-storage)
+to [Pangeo](https://medium.com/pangeo/continuously-extending-zarr-datasets-c54fbad3967d)
+are getting involved.
+
+<blockquote>We'd like to see a community agreement between the various parties on a minimal
+set of open formats covering a broad range of imaging modalities.</blockquote>
+
+We'd like to see the bioimaging community agree on set of open formats covering
+a broad range of imaging modalities. We need to reduce long-term cost of our
+domain's file formats and their variants. We want data users and producers
+to be able to ensure the long-term viability of their data.
```

Comment:
> We'd like to see the bioimaging community agree on set of open formats covering
a broad range of imaging modalities. We need to reduce long-term cost of our
domain's file formats and their variants. We want data users and producers
to be able to ensure the long-term viability of their data.

(a few less "we's")

We would like to see the Bioimaging community agree on set of open formats covering
a broad range of imaging modalities. This will help reduce the long-term cost of our domain's file formats and their variants, as well as ensure the long-term viability of our users and producers data.